### PR TITLE
Move product schema changes out of BMO extension

### PR DESCRIPTION
**Pull request by dylanwh**

### CODE REVIEW

File: Bugzilla/Config/GroupSecurity.pm [hunk 1]
```diff
@@ -84,6 +84,32 @@ sub get_param_list {
             name    => 'strict_isolation',
             type    => 'b',
             default => 0
+        },
+        {
+            name    => 'always_filleable_groups',
```
Comment (mohawk2) [inline Bugzilla/Config/GroupSecurity.pm]: Sorry for the drive-by review, was just having a quick skim. Is `filleable` spelled right there? Seems like that might cause problems down the line?

Comment (dylanwh) [reply Bugzilla/Config/GroupSecurity.pm]: Probably should have a better name, yeah.

Comment (dylanwh) [reply Bugzilla/Config/GroupSecurity.pm]: Hey, @mohawk2 if you would be so kind to take a look at bugzilla/harmony#25?

Some stuff is happening in that repo because today someone submitted like 17 PRs \o/
("harmony" is "bugzilla 6 should be based on bmo" project.)

Comment (mohawk2) [reply Bugzilla/Config/GroupSecurity.pm]: Done.

**Comment by dylanwh** [comment]
We're doing this in the upstream repo, bugzilla/harmony#31.
